notebooks/DATB/psf_grid_simulation_photometry/requirements.txt

```diff
@@ -0,0 +1,3 @@
+astropy>=3.0
```

Comment:
should this be 3.1?  Otherwise I'm confused why the envar above was changed to 3.1...

Reply:
It was just a test for webbpsf dependencies, but the recipe for webbpsf was actually broken (it does not require >=3.1).